### PR TITLE
CDL new command

### PR DESCRIPTION
**Pull request by Shellyber**
Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/27804

## Description
Added new command: cdl-get-file-data

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by Shellyber** [comment]
@kirbles19 Please review release notes and yml file

File: Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml [hunk 1]
```diff
@@ -1789,7 +1789,7 @@ script:
     - auto: PREDEFINED
       default: false
       description: First log time (<number> <time unit>. For example, 12 minutes,
-        7 days, 3 weeks).
+        7 days, 3 weeks)c
```
Comment (Itay4) [inline Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml:1792]: revert this

File: Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml [hunk 2]
```diff
@@ -1966,6 +1966,601 @@ script:
...
+      - spyware
+      - vulnerability
+      - file
+      - scan
+      - flood
+      - packet
+      - resource
+      - data
+      - url-content
+      - ' wildfire'
```
Comment (Itay4) [inline Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml:2230]:
```suggestion
      - wildfire
```

File: Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml [hunk 3]
```diff
@@ -1975,7 +2570,6 @@ script:
   script: '-'
   subtype: python3
   type: python
-  dockerimage45: demisto/python_pancloud_v2:1.0.0.10828
```
Comment (Itay4) [inline Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.yml:1978]: revert this

**Comment by ShirleyDenkberg** [comment]
@Shellyber Done

**Pull request merged**